submissions/alstep07/task-js-post-oop/index.js

```diff
@@ -0,0 +1,102 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+	constructor(species, name, gender, saying, legs, hands, friends) {
+		this.species = species;
+		this.name = name;
+		this.gender = gender;
+		this.saying = saying;
+      this.legs = legs;
+      this.hands = hands;
```

Comment:
This way all inhabitants have `hands` property even if they do not have any by natural design.
OOP and SOLID priciples help to design a system so that every entity has only those properties that are only required for them to have.

Reply:
@OleksiyRudenko Thank you for your review! I hope that in last commit I fixed all mistakes.